packages/core/stories/form-field/form-field.stories.tsx

```diff
@@ -205,6 +205,16 @@ export const WithValidation: ComponentStory<typeof FormField> = (props) => {
         <Input defaultValue="Input value" />
         <FormHelperText>Helper text</FormHelperText>
       </FormField>
+      <FormField validationStatus="error" {...props}>
+        <FormLabel>Error Form Field</FormLabel>
+        <Checkbox label="Option 1" />
+        <FormHelperText>Helper text</FormHelperText>
+      </FormField>
+      <FormField validationStatus="error" {...props}>
+        <FormLabel>Error Form Field</FormLabel>
+        <RadioButton label="Option 1" />
+        <FormHelperText>Helper text</FormHelperText>
+      </FormField>
```

Comment:
I think design can give some more realistic examples to add here, based on their research on forms

Reply:
agreed, waiting on specs from @bhoppers2008